### ISSUE TITLE
DotCover report command is missing an argument when calling buildParamsAndExecute 

### ISSUE THREAD

**Issue by TomasMorton** [fsharp/FAKE#2336]
### Description

At [Line 117](https://github.com/fsharp/FAKE/blob/master/src/app/Fake.DotNet.Testing.DotCover/DotCover.fs#L117), `buildParamsAndExecute` requires a `failBuild` parameter.
At [Line 175](https://github.com/fsharp/FAKE/blob/master/src/app/Fake.DotNet.Testing.DotCover/DotCover.fs#L175), the `report` function does not pass in a value for this parameter.  This leads to a mismatch between the [documentation page](https://fake.build/apidocs/v5/fake-dotnet-testing-dotcover.html) and the implementation in the image below.

See 
![image](https://user-images.githubusercontent.com/7551576/59472565-db35d200-8e0c-11e9-9c7a-790bf905a4a7.png).

### Repro steps

Try to use the `DotCover.report` function with the parameters required in the documentation.

### Expected behavior

The `failBuild` parameter is not required and is calculated internally.

### Actual behavior

The `failBuild` parameter is required and must be passed in manually.

### Known workarounds

Manually provide the parameter.

### Related information 

* Operating system: Windows 10, x64
* Branch: master
* .NET Runtime, CoreCLR or Mono Version: Dotnet core 2.2.103
* Indications of severity: 
* Version of FAKE (4.X, 5.X): 5.12.0, but master has this issue too.
* (if performance issue) Performance information, links to performance testing scripts

**Comment by matthid** [comment]
Can you please send a PR for this?

**Comment by TomasMorton** [comment]
I took a bit of time to identify where this parameter was added.  It was [here](https://github.com/fsharp/FAKE/commit/6f7205ce529d1fe72737d382b9730430dbd375da#diff-549cabe5e62ceaac0f3aefcccd4f0e60).  I assume the report function was simply forgotten.

Since I don't know what the correct parameter is, I will open a PR with the value set to false to "preserve" behaviour with the initial change, unless someone has a better idea.
